resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md

```diff
@@ -0,0 +1,315 @@
+  MobileCore.setLogLevel(LoggingMode.DEBUG);
+  try {
+    Analytics.registerExtension();
+    Identity.registerExtension();
+    MobileCore.start(new AdobeCallback() {
+      @Override
+      public void call(Object o) {
+        // add your app id from the "Environments" tab on Launch.
+        MobileCore.configureWithAppID("your-app-id");
+      }
+    });
+  } catch (InvalidInitException e) {
+    e.printStackTrace();
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+**Objective-C**
+
+```objective-c
+#import "ACPCore.h"
+#import "ACPAnalytics.h"
+#import "ACPIdentity.h"
+
+- (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPCore setLogLevel:ACPMobileLogLevelDebug];
+    [ACPAnalytics registerExtension];
+    [ACPIdentity registerExtension];
+    [ACPCore start:^{
+      // add your app id from the "Environments" tab on Launch.
+  		[ACPCore configureWithAppId:@"your-app-id"];
+    }];
+    return YES;
+}
+```
+
+**Swift**
+
+```swift
+import ACPCore
+import ACPAnalytics
+import ACPIdentity
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+  ACPCore.setLogLevel(ACPMobileLogLevel.debug)
+  ACPAnalytics.registerExtension()
+  ACPIdentity.registerExtension()
+  ACPCore.start(){
+  	ACPCore.configureWithAppId("your-app-id") 
+  }
+  return true
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+In-depth instructions can be seen at the [Analytics Readme](../../using-mobile-extensions/adobe-analytics/readme.md#add-analytics-to-your-app).
+
+##### Track App State and Track App Actions
+
+The V4 syntax and usage examples for these API are:
+
+{% tabs %}
+{% tab title="Android" %}
+
+```java
+// syntax
+public static void trackState(final String state, final Map<String, Object> contextData)
+
+// usage
+Analytics.trackState("MainPage", new HashMap<String, Object>() {{
+  put("firstVisit", true);
+}});
+```
+
+```java
+// syntax
+public static void trackAction(final String action, final Map<String, Object> contextData)
+  
+// usage
+Analytics.trackAction("linkClicked", new HashMap<String, Object>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+The AEP SDK's have moved the `trackAction` and `trackState` APIs to the MobileCore extension. In addition, the context data Map has been changed from `<String, Object>` to `<String, String>`: 
+
+```java
+// syntax
+public static void trackState(final String state, final Map<String, String> contextData)
+  
+// usage
+MobileCore.trackState("MainPage", new HashMap<String, String>() {{
+  put("firstVisit", "true");
+}});
+```
+
+```java
+// syntax
+public static void trackAction(final String action, final Map<String, String> contextData)
+  
+// usage
+MobileCore.trackAction("linkClicked", new HashMap<String, String>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
```

Comment:
Is this For V4? Not sure how tabs will look like, will the users know this part is for V4?